lib/views/feedbacks_slack_view.ex

```diff
@@ -1,7 +1,17 @@
 defmodule Aprb.Views.FeedbacksSlackView do
+  alias Aprb.Service.SentimentAnalysisService
+  
+  def prefix(event) do
+    emoji = event["properties"]["message"]
+      |>SentimentAnalysisService.sentiment_score
+      |>SentimentAnalysisService.sentiment_face_emoji
```

Comment:
I ❤️ pipe operations.

Minor, but seems like emoji could be its own function.

And I'm only imagining the screenshot, but do you need the `(...)`?